### ISSUE TITLE
Rule has failed to work: Rule attempted to report an ignored object in a violation

### ISSUE THREAD

**Issue by marlonrichert** [ygev/sketch-accessibility-assistant#26]
```
Rule sketch-accessibility-assistant/sketch-accessibility-assistant/aa-contrast-text produced an error: Rule attempted to report an ignored object in a violation @
report@
n@
@
generatorResume@[native code]
@
Promise@[native code]
@
@
generatorResume@[native code]
@
Promise@[native code]
ue@
@
asyncFunctionResume@[native code]
@[native code]
promiseReactionJobWithoutPromise@[native code]
```
